src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,321 @@
+            {/* Safe App details */}
+            {safeAppDetails && (
+              <DetailsContainer>
+                <SafeIcon src={safeAppDetails.iconUrl} />
+                <DescriptionContainer>
+                  <SafeAppTitle size={'sm'}>{safeAppDetails.name}</SafeAppTitle>
+                  <div>{safeAppDetails?.description}</div>
+                </DescriptionContainer>
+              </DetailsContainer>
+            )}
+            <Separator />
+
+            {/* Available chains */}
+            {showAvailableChains && (
+              <>
+                <ChainLabel size="lg">Available networks</ChainLabel>
+                <ChainsContainer>
+                  {availableChains.map((chainId) => (
+                    <div key={chainId}>
+                      <NetworkLabel networkInfo={getChainById(chainId)} />
+                    </div>
+                  ))}
+                </ChainsContainer>
+                <Separator />
+              </>
+            )}
+
+            <ActionsContainer>
+              <UserSafeContainer>
+                <Title size={'xs'}>Use the dapp with your Safe!</Title>
+                {isWalletConnected ? (
+                  <>
+                    {userSafe ? (
+                      <div>not implemented</div>
+                    ) : (
+                      <>
+                        {/* Create new Safe */}
+                        <BodyImage>
+                          <Img alt="Vault" height={92} src={SuccessSvg} />
+                        </BodyImage>
+
+                        <Button
+                          data-testid={'create-new-safe-link'}
+                          size="lg"
+                          color="primary"
+                          variant="contained"
+                          component={Link}
+                          to={OPEN_SAFE_ROUTE}
+                        >
+                          <Text size="xl" color="white">
+                            Create new Safe
+                          </Text>
+                        </Button>
+                      </>
+                    )}
+                  </>
+                ) : (
+                  <>
+                    {/* Connect your Wallet */}
+                    <StyledProvider>
+                      <Provider
+                        info={<ProviderDisconnected />}
+                        open={open}
+                        toggle={toggle}
+                        render={(providerRef) =>
+                          providerRef.current && (
+                            <Popper
+                              anchorEl={providerRef.current}
+                              style={{ zIndex: 1301 }}
+                              open={open}
+                              placement="bottom"
+                              popperOptions={{ positionFixed: true }}
+                            >
+                              <ClickAwayListener mouseEvent="onClick" onClickAway={clickAway} touchEvent={false}>
```

Comment:
`onClick` is the default prop.

```suggestion
                              <ClickAwayListener onClickAway={clickAway} touchEvent={false}>
```

Reply:
done!